### ISSUE TITLE
Cannot close html video player once started

### ISSUE THREAD

**Issue by davidubi73** [nextcloud/server#5505]
When playing an uploaded video file, there is a darker overlay over the web interface, and the video stays in the foreground, but there is no control button whatsoever to close the player screen and the only way to go back to the user interface is using the url bar in the browser to reload the home page of my nextcloud instance.

Server configuration: Nextcloud 12.0.0, Debian 9 amd64, Apache 2.4.25, PHP 7.0.19-1, MariaDB 10.1.23-9
Client: Windows 7 32bit, Browser: Firefox 52 ESR; Debian 9 amd64, Browser: Firefox 52 ESR

**Comment by antonanders** [comment]
i have the same issue.
Fresh Nextcloud 12.0.0 instance from official docker image, Traefik as reverse proxy.
Browser:. Vivaldi 10.867.38 (Stable channel) (64-Bit), Arch Linux

**Comment by RainerEmrich** [comment]
That's a duplicate of #4968

**Comment by maprambo** [comment]
Not really. In addition, the video player doesn't show it's controls for me, it's the browser. May be a fault of mine, what do you observe, @davidubi73 ?

**Comment by moltenkaizen** [comment]
@davidubi73 I had the same issue.

Checking out the Web Console shows an error in viewer.js:
<pre>
Uncaught ReferenceError: videojs is not defined
    at Object.show (viewer.js?v=651b82c…-12:40)
    at Object.<anonymous> (viewer.js?v=651b82c…-12:84)
    at Object.<anonymous> (core.js?v=651b82c…-12:2)
    at j (core.js?v=651b82c…-12:2)
    at Object.fireWith [as resolveWith] (core.js?v=651b82c…-12:2)
    at x (core.js?v=651b82c…-12:4)
    at XMLHttpRequest.<anonymous> (core.js?v=651b82c…-12:4)
</pre>

It turns out there's a bug in line 11 of viewer.js of the Nextcloud videoplayer

Nextcloud videoplayer with the bug is in the zip archive on https://nextcloud.com/install/#instructions-server
I just downloaded the archive that is live right now to verify.
https://download.nextcloud.com/server/releases/nextcloud-12.0.0.zip

It's already been reported and patched:
https://github.com/nextcloud/files_videoplayer/issues/25

The video player was patched on May 26 with this commit (12.0.0 released May 22):
https://github.com/nextcloud/files_videoplayer/commit/cd0a72e67052abbd163e3bd8bc641abf0ac25638

It would be a good idea if someone could fix the archive that is currently live. Though it sounds like it will be patched in 12.0.1 and released at the end of June.

To sum up:
You could just fix that one line in the js, install the latest version of the nextcloud videoplayer, or wait for 12.0.1

**Comment by davidubi73** [comment]
Thanks, moltenkaizen! I just changed line 11 in (yournextcloudroot)/apps/files_videoplayer/js/viewer.js 
`return $.getScript(scriptPath)
`
is changed to...
`return $.getScript(scriptPath, function (xhr) {eval(xhr);});`
and it works like a charm. Hope the fix arrives soon for everybody else with the 12.0.1 update.

**Comment by RainerEmrich** [comment]
Confirmed, fix works for me.

**Comment by yamlfd** [comment]
Never use a x.0 get's a new light with nextcloud...

**Comment by MorrisJobke** [comment]
Fixed in 12.0.1

**Comment by crickxson** [comment]
Using the 12.0.3, and no close button... Thx
